nilearn/decoding/tests/test_decoder.py

```diff
@@ -1221,7 +1223,7 @@ def test_decoder_adjust_screening_greaterthan_mask_surface(
 
 
 @pytest.mark.parametrize("mask", [None, SurfaceMasker()])
-@pytest.mark.parametrize("decoder", [_BaseDecoder, Decoder, DecoderRegressor])
```

Comment:
removing _BaseDecoder from some test parametrization: it should be enough to test it through its children classes